www/caddy/src/opnsense/mvc/app/controllers/OPNsense/Caddy/forms/dialogHandle.xml

```diff
@@ -172,4 +164,53 @@
         <style>style_tls style_reverse_proxy</style>
         <help><![CDATA[Enable or disable NTLM. Needed to reverse proxy an Exchange Server. Warning: NTLM has been deprecated by Microsoft. This option will stay for as long as the optional http.reverse_proxy.transport.http_ntlm module can be compiled without errors.]]></help>
     </field>
+    <field>
+        <type>header</type>
+        <label>Load Balancing</label>
+        <advanced>true</advanced>
+    </field>
+    <field>
+        <id>handle.PassiveHealthFailDuration</id>
+        <label>Upstream Fail Duration (s)</label>
+        <type>text</type>
+        <style>style_reverse_proxy</style>
+        <hint>off</hint>
+        <help><![CDATA[fail_duration enables a passive health check when multiple destinations in "Upstream Domain" are set. It is a value that defines how long to remember a failed request. A duration of 1 or more seconds enables passive health checking. A reasonable starting point might be 30s to balance error rates with responsiveness when bringing an unhealthy upstream back online.]]></help>
+        <advanced>true</advanced>
+    </field>
+    <field>
+        <id>handle.lb_policy</id>
+        <label>Load Balance Policy</label>
+        <type>dropdown</type>
+        <style>style_reverse_proxy</style>
+        <help><![CDATA[lb_policy is the name of the load balancing policy, along with any options. For policies that involve hashing, the highest-random-weight (HRW) algorithm is used to ensure that a client or request with the same hash key is mapped to the same upstream, even if the list of upstreams change.]]></help>
+        <advanced>true</advanced>
+    </field>
+    <field>
+        <id>handle.lb_retries</id>
+        <label>Load Balance Retries</label>
+        <type>text</type>
+        <style>style_reverse_proxy</style>
+        <hint>off</hint>
+        <help><![CDATA[lb_retries is how many times to retry selecting available backends for each request if the next available host is down. If lb_try_duration is also configured, then retries may stop early if the duration is reached. In other words, the retry duration takes precedence over the retry count.]]></help>
+        <advanced>true</advanced>
+    </field>
+    <field>
+        <id>handle.lb_try_duration</id>
+        <label>Load Balance Try Duration (s)</label>
+        <type>text</type>
+        <style>style_reverse_proxy</style>
+        <hint>0 </hint>
```

Comment:
This is not a typo.

A hint with only 0 will not be displayed. Adding a whitespace fixes that.

Reply:
“0” is a strangely irrelevant default 🤷‍♂️

Reply:
In this case its not since empty can mean "off" or actually 0 seconds in this one case.

I guess I just leave it like this with a whitespace xD

Reply:
![image](https://github.com/user-attachments/assets/97a401cd-d544-4325-a687-86e994701322)

Reply:
https://github.com/opnsense/core/commit/1dcec96bcb

```suggestion
        <hint>0</hint>
```

;)